components/blitz/resources/omero/Constants.ice

```diff
@@ -214,14 +214,6 @@ module omero {
      **/
     module analysis {
 
-       /** namespaces related to the FLIM analysis. **/
```

Comment:
For me it's not as much about the being able to use the scripts as that changing these files can break someone's compilation (Java, C++)